pkg/flagset/flagset.go

```diff
@@ -148,5 +148,72 @@ func ServerWithConfig(cfg *config.Config) []cli.Flag {
 			EnvVar:      "PHOENIX_WEB_CONFIG",
 			Destination: &cfg.Phoenix.Path,
 		},
+		&cli.StringFlag{
+			Name:        "web-config-server",
+			Value:       "http://localhost:9135",
+			Usage:       "Server URL",
+			EnvVar:      "PHOENIX_WEB_CONFIG_SERVER",
+			Destination: &cfg.Phoenix.Config.Server,
+		},
+		&cli.StringFlag{
+			Name:        "web-config-theme",
+			Value:       "owncloud",
+			Usage:       "Theme",
+			EnvVar:      "PHOENIX_WEB_CONFIG_THEME",
+			Destination: &cfg.Phoenix.Config.Theme,
+		},
+		&cli.StringFlag{
+			Name:        "web-config-version",
+			Value:       "0.1.0",
+			Usage:       "Version",
+			EnvVar:      "PHOENIX_WEB_CONFIG_VERSION",
+			Destination: &cfg.Phoenix.Config.Version,
+		},
+		/*
+			 // TODO somehow cast to *StringSlice ... which is a []string
+			 // or find a way to provide a default value
+			&cli.StringSliceFlag{
+				Name:   "web-config-apps",
+				Value:  &cfg.Phoenix.Config.Apps,
+				Usage:  "Version",
+				EnvVar: "PHOENIX_APPS",
+			},
+		*/
+		// TODO EXTERNAL APPS?
```

Comment:
you need to declare apps:

`apps := cli.StringSlice(cfg.Phoenix.Config.Apps)`

and then these can be passed for to the structure

```go
&cli.StringSliceFlag{
	Name:   "web-config-apps",
	Value:  &apps,
	Usage:  "Version",
	EnvVar: "PHOENIX_APPS",
},
```

Reply:
That will copy the []string ... the code will not pick up the apps

Reply:
wat? no, that uses type conversion to make cfg.Phoenix.Config.Apps (type []string) into a `cli.StringSlice` but stores the values on it. Unless they're overridden elsewhere, this works just fine, I have a small example here:

```go
package main

import (
	"fmt"
	"os"

	"github.com/urfave/cli"
)

func main() {
	apps := cli.StringSlice([]string{"1", "2"})
	cli.HelpFlag = cli.StringSliceFlag{
		Name:   "web-config-apps",
		Value:  &apps,
		Usage:  "Version",
		EnvVar: "PHOENIX_APPS",
	}

	fmt.Println(cli.HelpFlag.String())
	(&cli.App{}).Run(os.Args)
}
```